news/92.bugfix

```diff
@@ -0,0 +1 @@
+Forces the scale uid to be the same, regardless of the scale parameter being passed. @wesleybl
```

Comment:
I understand what you mean, but I think this can be misinterpreted.  How about the following:

```suggestion
If width and height are given, ignore the scale parameter when determining the unique id of a scale. @wesleybl
```